src/transformers/models/xmod/modeling_xmod.py

```diff
@@ -793,7 +793,6 @@ class XmodModel(XmodPreTrainedModel):
 
     """
 
-    # Copied from transformers.models.bert.modeling_bert.BertModel.__init__ with Bert->Xmod
```

Comment:
For all the removed `# Copied from`, I would rather add a comment that this is adapted from bert, and the copied from could be added back once these models support SDPA.